ome/omero-figure
# Maintain aspect ratio Button

This PR adds an extra button to control the aspect ratio of the panels, when the user edits the width/height of the image.

To Test:
1) Test if toggling the button changes the colour to red(inactive) and green(active) states.
2) When the button is red, edit width or height of the image. This will just change the width of height of the image.
3) When the button is green, edit width or height of the image. Please check the width/height ratio (aspect ratio) of the panel before editing. Post editing the width/height of the panel, the height/width of the image should automatically get calculated and adjusted. The resultant panel aspect ratio, should be exactly the same as the previous aspect ratio.
4) Check if the tooltip on the button makes sense.

And as always code review would be useful as well.
The branch includes the code-layout branch as well, and the PR could technically be considered as an independent review of the code-layout PR from @will-moore 
@will-moore @jburel
